docs(gitlab): add permission note for group level access token (#26583)
renovatebot/renovate: lib/modules/platform/gitlab/readme.md

First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account.

> Note: If you are using a Group Level Access Token, the token must be Developer role or higher to create issues and merge requests. The account must be Maintainer or higher to perform Auto Merge. 

Reviewer: ```suggestion
If you are using a Group access token, the token must have Developer role or higher permissions in order to create issues and merge requests.
The token must have Maintainer permissions in order to perform Automerge. 
```
